extension/bridge/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "mozillavpnnp"
+version = "0.1.0"
```

Comment:
This is probably fine for testing, but we should find some way for this to track the version of the main VPN client app.